app/models/miq_schedule_worker.rb

```diff
@@ -4,11 +4,7 @@ class MiqScheduleWorker < MiqWorker
   require_nested :Jobs
   require_nested :Runner
 
-  self.check_for_minimal_role = false
-  self.workers                = lambda {
-    return MiqServer.minimal_env_options.include?('schedule') ? 1 : 0 if MiqServer.minimal_env?
-    return 1
-  }
+  self.workers = 1
```

Comment:
I'm not sure about this one...I think can remove it altogether, since you can't really change the worker count of miq_schedule?  @jrafanie Thoughts?

Reply:
Yeah, there's no harm of having the schedule running in whatever becomes of a spartan procfile

Reply:
@jrafanie, Ok, I can't remove it, technically, because then changing the value in settings will then cause MiqScheduleWorker.workers to return that value.  Even though the server role defines the max_concurrent as 1, I think that only means that only 1 server can have the _role_ concurrently, but within that 1 server if would run N workers.

Note that the other option here is that we can set `self.maximum_workers_count = 1`.  This value acts as a max clamp on the configured number, which would then allow 0 workers, which I don't think we want either.  The other reason I don't like that option is because the only reason it exists it to clamp the max number of web server workers per server (which now that I think about it sounds like a bad idea for podified :| ).